indexer/feature_utils.cpp

```diff
@@ -503,4 +503,41 @@ string FormatElevation(string_view elevation)
   return {};
 }
 
+constexpr char const * kWlan = "wlan";
+constexpr char const * kWired = "wired";
+constexpr char const * kTerminal = "terminal";
+constexpr char const * kYes = "yes";
+constexpr char const * kNo = "no";
+
+string DebugPrint(Internet internet)
```

Comment:
Can this DebugPrint return std::string_view?

Reply:
I think other parts of the code expect DebugPrint to return string. CI checks are failing.